src/lib/textToCad.ts

```diff
@@ -197,30 +200,28 @@ export async function submitAndAwaitTextToKclSystemIO({
 
       const TRUNCATED_PROMPT_LENGTH = 24
       // Only add the prompt name if it is a preexisting project
-      newFileName = `${value.prompt
+      const subDirectoryAsPromptName = `${value.prompt
         .slice(0, TRUNCATED_PROMPT_LENGTH)
         .replace(/\s/gi, '-')
         .replace(/\W/gi, '-')
-        .toLowerCase()}${FILE_EXT}`
-
-      // If the project is new generate a main.kcl
-      if (isProjectNew) {
-        newFileName = PROJECT_ENTRYPOINT
-      }
+        .toLowerCase()}`
 
       if (isDesktop()) {
-        // We have to preemptively run our unique file name logic,
-        // so that we can pass the unique file name to the toast,
-        // and by extension the file-deletion-on-reject logic.
-        newFileName = getNextFileName({
-          entryName: newFileName,
-          baseDir: projectName,
-        }).name
-
+        if (!isProjectNew) {
```

Comment:
Like we mentioned earlier, we should follow up with a PR to make the project name the truncated prompt name (made unique) if `isProjectNew`